application/yii

```diff
@@ -8,8 +8,12 @@
  * @license http://www.yiiframework.com/license/
  */
 
-defined('YII_DEBUG') or define('YII_DEBUG', true);
-defined('YII_ENV') or define('YII_ENV', 'dev');
+use Sil\PhpEnv\Env;
+
+require(__DIR__ . '/vendor/autoload.php');
```

Comment:
The composer autoload file is already being required below (new line 22). Can it be removed here? (or there)

Reply:
Good observation. I suppose it would be better to remove the later one, but I'm not as comfortable with PHP includes as I used to be.

Reply:
You'll need it included before you use the `Env` class, so either just keep the top one, or use PHP's built-in functions for getting the env. var. and keep the lower require. My suggestion.